proto/buf/alpha/registry/v1alpha1/repository_branch.proto

```diff
@@ -31,6 +31,8 @@ message RepositoryBranch {
   bool is_main_branch = 4;
   // The last update time of the branch.
   google.protobuf.Timestamp last_update_time = 5;
+  // The last update author of the branch.
```

Comment:
What is the format of this field? What is being stored? What is an author? This needs documentation.

Reply:
Thanks, I'll add more detailed documentation. It would contain the name of the author of the most recent branch label. An author can be a committer in the case if a branch is updated by a git commit, or a BSR user if it is updated by a new BSR commit.  @unmultimedio @doriable, please correct me if I'm wrong.

Reply:
Based on offline chat with @unmultimedio and @doriable, we'll rename this field to `last_update_user_id` which will contain the BSR user id who updated this branch last.